qiime2/sdk/action.py

```diff
@@ -342,6 +344,8 @@ def _build_numpydoc(self):
         parameters = self._build_section("Parameters", sig.signature_order)
         returns = self._build_section("Returns", sig.outputs)
 
+        # TODO: include Usage-rendered examples here
```

Comment:
It seems like you have all you need to make this happen, it would be pretty cool to add.